Tasks/SonarQubePostTest/PRCA/PostComments-Server.ps1

```diff
@@ -226,8 +186,9 @@ function FetchDiscussionComments
     
     foreach ($discussionThread in $discussionThreads)
     {
-        $commentsFromThread = [PsWorkarounds.Helper]::GetComments($script:discussionClient, $discussionThread.DiscussionId)
-        
+        $request = InvokeByReflection  $script:discussionClient "GetCommentsAsync" @([Int]) @($discussionThread.DiscussionId)
```

Comment:
Don't they have an API with multiple comments? I imagine it would be more efficient use of resources...